card/mod/01_core/chunk/include.rb

```diff
@@ -5,7 +5,7 @@
 module Card::Chunk
   class Include < Reference
     cattr_reader :options
-    @@options = ::Set.new [ :inc_name, :inc_syntax, :view, :items, :type, :size, :title, :hide, :show, :structure ]
+    @@options = ::Set.new [ :inc_name, :inc_syntax, :view, :items, :type, :size, :title, :hide, :show, :structure, :unlabeled ]
```

Comment:
could we do this as hide: set_label?  This seems like a fairly narrow case for new nest syntax.  As you've probably noticed, I try to keep the bar for new nest syntax high, because we're soon going to introduce a gui nest editor, and I want it to be very streamlined.